change/@react-native-windows-cli-7f4e85c8-bb7a-450d-ab21-2959561ea5f3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This doesn't appear to be necessary? The package wasn't touched in this integrate.